optimize/backend/src/main/java/io/camunda/optimize/service/importing/page/AllEntitiesBasedImportPage.java

```diff
@@ -1,30 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Camunda License 1.0. You may not use this file
- * except in compliance with the Camunda License 1.0.
- */
-package io.camunda.optimize.service.importing.page;
-
-public class AllEntitiesBasedImportPage implements ImportPage {
```

Comment:
Readers were generally supposed to be part of [feature removal PRs](https://github.com/camunda/camunda-optimize/issues/13211) to control PR scope were possible. Looking at these its not really clear to me what/if this `AllEntitiesBasedImportIndexDto` is relevant anymore, looks like its not. Removal could then also cover the respective ImportRepository helpers. I think it this case its so tightly related to import and a small enough change I dont mind adding it in this PR instead of the other ticket